eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -95,6 +95,19 @@ jobs:
         MapFileArtifactSuffix: 'win-x86-rel-noprc-nolog'
         BuildType: Release
 
+      Windows2022_VS2022_X86_Release_MapFiles_UnitTests:
+        Pool: azsdk-pool-mms-win-2022-general
+        OSVmImage: MMS2022
+        vcpkg.deps: 'cmocka'
+        VCPKG_DEFAULT_TRIPLET: 'x86-windows-static'
+        build.args: '-DPRECONDITIONS=OFF -DUNIT_TESTING=ON -DLOGGING=OFF -DADDRESS_SANITIZER=ON'
+        CMAKE_GENERATOR: 'Visual Studio 17 2022'
+        CMAKE_GENERATOR_PLATFORM: Win32
+        AZ_SDK_C_NO_SAMPLES: 'true'
+        PublishMapFiles: 'true'
```

Comment:
This is causing a collision. Remove this line and change the name to `Windows2022_VS2022_X86_Release_UnitTests`